lib/embed/embed.dart

```diff
@@ -112,6 +112,8 @@ class PlaygroundMobile {
     String strong = url.queryParameters['strong'];
     if (_parseStrongModeParam(strong)) {
       (querySelector('#strongmode') as InputElement).checked = true;
+    } else {
+      (querySelector('#strongmode') as InputElement).checked = false;
```

Comment:
could do: `(querySelector('#strongmode') as InputElement).checked = _parseStrongModeParam(strong)`

Reply:
i.e., not need for an if statement, if you're always going to assign